Duplicati/Library/Main/Operation/ListFilesHandler.cs

```diff
@@ -51,7 +51,10 @@ public void Run(IEnumerable<string> filterstrings = null, Library.Utility.IFilte
                         IEnumerable<Database.LocalListDatabase.IFileversion> files;
                         if (m_options.ListFolderContents)
                         {
-                            files = filesets.SelectFolderContents(filter);
+                            if (string.Equals(Environment.GetEnvironmentVariable("EXPERIMENTAL_DBLIST_DUPLICATI") ?? string.Empty, "1"))

```

Comment:
The process probably varies and/or is fluid. One example of recent process try is at
[Preview of security and bugfix release Canary 105](https://forum.duplicati.com/t/preview-of-security-and-bugfix-release-canary-105/15925), and testers are very welcome...

IIRC of the three experiments, one made it out. Maybe others will, sometime later.
The goal was to get to Beta rather quickly to give the biggest seemingly safe fixes.

From a terminology point of view, the above was the transition from preview (which didn't used to happen) to Canary.
That's an official release of the first public view (that includes automatic updates) to those willing to run bleeding-edge.
Sometimes one needs to because of some bug fix that's needed (much so for 105), but it can certainly hold surprises...
Once Canary goes, it's basically progressive public qualification of frozen function, and finally hits Beta and stops there.

The arguments for and against a more complex GitHub branching model have been in the forum. There are drawbacks (especially as release time approaches) to a simple model of limited flow to master, but people disliked complex models.

I'm not sure if simple will hold up under the weight of leaving .NET Framework. May I encourage you to join the forum?
The developer category is, IMO, a good place for developer discussions, though I gave some info here to start this up...
There's another technical talk brewing in a GitHub issue among non-technical people that I'd also like to get re-homed.

Thanks for your interest. I'm just trying to get the discussions in the right place.